### ISSUE TITLE
[API] Errors/Cannot find right section

### ISSUE THREAD

**Issue by Gardiac** [DRE2N/FactionsOne#37]
Hey
So I made a plugin long time ago that is actually pretty cool addon and it worked fine with any factions except now I cannot find these in your F_One :/
Could you help please?
http://prntscr.com/gs9dt2
http://prntscr.com/gs9e9g (Lately I found out that first Board could be done without getting instance)
http://prntscr.com/gs9ep8

**Comment by Sataniel98** [comment]
Massivecraft called all singleton instance fields "i".

**Comment by Gardiac** [comment]
Could you make example of let's say first screenshot?

**Comment by Sataniel98** [comment]
FPlayers.i.get()... You'll need to have a look at the source code. FactionsUUID for example renamed a lot of methods.

**Comment by Gardiac** [comment]
There are no such methods I've also looked in to the src?

**Comment by Sataniel98** [comment]
Yes there are. Some of the methods are inherited from EntityCollection, though.

**Comment by Gardiac** [comment]
could you please write all of them down? <3

**Comment by Sataniel98** [comment]
For the cases you mentioned, that should be: "FPlayers.i.get(player).getFaction()". I won't spoonfeed you all getters in the entire plugin since it's actually quite simple if you understand the principle.

FactionsUUID | original

Factions, FPlayers = Factions, FPlayers

getInstance() = i

getByPlayer(Player), getBySomethingElse(SomethingElse) = get(Player), get(SomethingElse)
